packages/components/src/components/animations/DeviceAnimation.tsx

```diff
@@ -105,7 +105,7 @@ export const DeviceAnimation = forwardRef<HTMLVideoElement, DeviceAnimationProps
                     >
                         <source
                             src={resolveStaticPath(
-                                `videos/device/trezor_${DeviceModelInternal.T1B1.toLowerCase()}_${type.toLowerCase()}${theme}.webm`,
+                                `videos/device/trezor_${DeviceModelInternal.T1B1.toLowerCase()}_${type.toLowerCase()}${themeSuffix}.webm`,
```

Comment:
Ack. This could be caught by eslint rule `no-base-to-string`, I'll try playing around with that.